lib/OpenQA/WebAPI.pm

```diff
@@ -216,9 +216,9 @@ sub startup {
 
     # Favicon
     $r->get('/favicon.ico' => sub { my $c = shift; $c->render_static('favicon.ico') });
+    $r->get('/index' => [format => ['html', 'json']])->to('main#index');
```

Comment:
My initial version only supported /index.json - but I wanted to also support index.html to avoid confusion